src/fmeval/model_runners/extractors/jumpstart_extractor.py

```diff
@@ -56,15 +56,19 @@ def __init__(
             self._sagemaker_session.boto_region_name,
         )
         util.require(DEFAULT_PAYLOADS in model_spec_key, f"Model: {jumpstart_model_id} is not supported at this time")
-        output_jmespath_expressions = compile_jmespath(GENERATED_TEXT_JMESPATH_EXPRESSION).search(
-            model_spec_key[DEFAULT_PAYLOADS]
-        )
-        util.require(
-            output_jmespath_expressions,
-            f"Jumpstart model {jumpstart_model_id} is likely not supported. If you know it is generates text, "
```

Comment:
Let's reword this to something like: "Default payload for Jumpstart could not be constructed. Please configure output and log probability instead."